### PR TITLE
Guide to retrieve ske outgoing cluster ip

### PR DESCRIPTION
**Pull request by h3adex**
I wrote some documentation on how to gather the outgoing IP address of the SKE cluster for my own reference. I believe this could be helpful for other developers as well. Please feel free to read through it and share any feedback if something is not clear.

Additionally, I suggest we link this guide to the following issue for better visibility: [Issue #102](https://github.com/stackitcloud/terraform-provider-stackit/issues/102).

Thank you! :-)

### CODE REVIEW

**Review by saurabh2590 — APPROVED** [review]
Looks good to me and we were able to use the instructions to look up router IP.

File: docs/guides/outgoing_ske_ip.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+page_title: "Retrieving SKE outgoing IP address with Terraform"
+---
+# Retrieving SKE outgoing IP address with Terraform
+
+To retrieve the outgoing IP address of your Kubernetes cluster, you have two options based on your initial SKE setup.
```
Comment (DiogoFerrao) [inline docs/guides/outgoing_ske_ip.md:6]:
```suggestion
To retrieve the outgoing IP address of your STACKIT Kubernetes Engine (SKE) cluster, you have two options based on your initial SKE setup.
```

File: docs/guides/outgoing_ske_ip.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Default Setup
+
+If you haven't configured any organisational network settings, you can use the default setup. It is necessary to use the Terraform OpenStack provider in conjunction with the STACKIT provider.
```
Comment (DiogoFerrao) [inline docs/guides/outgoing_ske_ip.md:10]:
```suggestion
If you haven't configured any organizational network settings, you can use the default setup. It is necessary to use the Terraform OpenStack provider in conjunction with the STACKIT provider.
```

Comment (DiogoFerrao) [reply docs/guides/outgoing_ske_ip.md:10]: We should stick to the American English spelling

File: docs/guides/outgoing_ske_ip.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+locals {
+  cluster_outgoing_ip = data.openstack_networking_router_v2.router.external_fixed_ip.0.ip_address
+}
+```
+
+## Organizational SNA Setup
+
+If your project is within an organisational SNA, you can attach a `stackit_network` to the ske cluster:
```
Comment (DiogoFerrao) [inline docs/guides/outgoing_ske_ip.md:31]:
```suggestion
If your project is within an organizational STACKIT Network Area (SNA), you can attach a `stackit_network` to the SKE cluster:
```

File: docs/guides/outgoing_ske_ip.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  }
+}
+
+locals {
+  cluster_outgoing_ip = stackit_network.ske_network.public_ip
+}
+```
+
+In both examples, the attribute `cluster_outgoing_ip` will provide the outgoing IP address of your cluster.
+The specific configuration depends on whether your setup is within an organisational SNA or a default setup.
```
Comment (DiogoFerrao) [inline docs/guides/outgoing_ske_ip.md:58]:
```suggestion
The specific configuration depends on whether your setup is within an organizational SNA or a default setup.
```

**Comment by DiogoFerrao** [comment]
Hey @h3adex, sorry for the delayed response to your PR.

Firstly I would like to thank you for this contribution! We always welcome this sort of help and this guide is definitely useful for our users.

Also, I really like the writing on it! I will leave some comments below with some details but it looks great overall!

Once these comments are addressed, we can merge the PR.

**Comment by muslashwhy** [comment]
Hi @DiogoFerrao

I've applied all of you comments. I've amended all changes to the latest commit.

Thank you

**Comment by h3adex** [comment]
Hi @DiogoFerrao

I've applied all of you comments. I've amended all changes to the latest commit.

Thank you

**Review by joaopalet — APPROVED** [review]

**Comment by joaopalet** [comment]
Hi @h3adex,

Looks good to me, thanks for the contribution! Can we merge the PR?

**Comment by h3adex** [comment]
Yes. Let's go :-)

**Comment by h3adex** [comment]
@DiogoFerrao 

We should also inform this [issue](https://github.com/stackitcloud/terraform-provider-stackit/issues/102) about this PR.

**Pull request merged**